### ISSUE TITLE
KubeJS request

### ISSUE THREAD

**Issue by deftesthawk** [yanny7/YTech#14]
I'm so thankful that you provide KubeJS support, I have a couple things that I think would be great additions if you're okay with adding them.
1. Add a chance value variable to the block hit recipe it could also have a optional secondary output variable for if it fails and if one isn't provided it wouldn't give anything.
2. The ability to change/remove existing recipes for all recipe types.

**Comment by deftesthawk** [comment]
Just realized that the tree stump doesn't have support so I added it to the list

**Comment by yanny7** [comment]
Tree stump is missing in Wiki but it's implemented. For workspace crafting it's impossible to set tool.

**Comment by deftesthawk** [comment]
Okay, thanks. I'll see if I can work out how todo the tree stump recipes.

**Comment by yanny7** [comment]
Wiki was updated, check https://github.com/yanny7/YTech/wiki/KubeJS#chopping

**Comment by yanny7** [comment]
I think that changing/removing recipes by type it's up to KubeJS, not mod integration
```
event.remove({ type: 'ytech:chopping' })
```

**Comment by deftesthawk** [comment]
I just figured that out too xD

**Comment by deftesthawk** [comment]
Added a new request, I tried testing to see if there was a way of adding an output count to certain recipes but couldn't find a way so I'm assuming there's not.

**Comment by yanny7** [comment]
This is how you set output count
```
event.recipes.ytech.chopping('5x minecraft:diamond', Item.of("minecraft:oak_log"), Item.of("ytech:flint_axe"))//.hitCount(10);
```
please check KubeJS doc first

**Comment by deftesthawk** [comment]
Sorry, I missed that part on the wiki. Thought it would have been. 
event.recipes.ytech.chopping('minecraft:diamond, 2', Item.of("minecraft:oak_log"), Item.of("ytech:flint_axe"))//.hitCount(10);

I’ll make sure to double check in future, sorry again

**Comment by deftesthawk** [comment]
If it's possible maybe adding .hitCount(4) to some of the other recipes types, for example it would feel immersive to hammer the item multiple times on the anvil or have to hit an item on a block multiple times.

I also noticed that the golden shears name is display as "item.ytech.golden_shears" I created a resource pack with the lang file to fix it on my end but thought it might be worth mentioning

**Comment by yanny7** [comment]
This is already available in v0.9.0 released two hours ago :)

**Comment by deftesthawk** [comment]
Thank you! Looks like a cool update :)

**Comment by deftesthawk** [comment]
Not sure if this was intentional but the stone mortar and pestle requires #minecraft:planks xD

**Comment by yanny7** [comment]
Yes it's intentional, but maybe it should be renamed to wooden mortar and pestle... Stone was gated, so I had to replace stone with something else.

**Comment by deftesthawk** [comment]
Ah that makes sense, yeah it seemed out of place with the new recipes due to all but that one requiring the material they’re named after. Wooden mortar and pestle would definitely fit the recipe
